### PR TITLE
fix: caching problem with PWA and login providers

### LINKED ISSUE #2850: Fix 3rd party auth 404 redirect in combination with PWA
If PWA feature is enabled, the 3rd party auth leads to a 404 page instead of the 3rd party login pages. We don't know the exact cause and this needs to be fixed

### PR DESCRIPTION
**Pull request by Resaki1**
# [Demo](https://development.scrumlr.fra.ics.inovex.io)

## Description

With the new PWA features, the external login providers weren't working properly. This has been fixed. Closes #2850 

## Changelog

- service-worker has been re-activated
- URIs including `/api/login/` are not cached by the service-worker anymore

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] The light- and dark-theme are both supported and tested
- [x] The design was implemented and is responsive for all devices and screen sizes
- [x] The application was tested in the most commonly used browsers (e.g. Chrome, Firefox, Safari)

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -41,4 +42,4 @@ root.render(
 store.dispatch(Actions.initApplication());
 
 // Disable the service worker for now since there are known issues where our third party authentication services cannot be used
-// serviceWorkerRegistration.register();
+serviceWorkerRegistration.register();
```
Comment (CronJorian) [inline src/index.tsx:45]:
```suggestion
register();
```

File: src/index.tsx [hunk 2]
```diff
@@ -41,4 +42,4 @@ root.render(
 store.dispatch(Actions.initApplication());
 
 // Disable the service worker for now since there are known issues where our third party authentication services cannot be used
```
Comment (CronJorian) [inline src/index.tsx:44]: If it's not disabled anymore, this comment is useless

**Review by CronJorian — APPROVED** [review]
Works on my machine 💪 
Left some comments, but none are required

File: .env [hunk 3]
```diff
@@ -1,3 +1,4 @@
 PUBLIC_URL=/
 REACT_APP_PUBLIC_URL=/
 REACT_APP_VERSION=$npm_package_version
+
```
[the remark this answers is not in the document]
Comment (Resaki1) [reply .env:4]: it's only one newline at the end as far as I can tell

**Pull request merged**